### ISSUE TITLE
Air Photos

### ISSUE THREAD

**Issue by emilyodean** [NBMG-UNR/resourcespace-migration#2]
We have a TON of air photos which all have different indices. We need to come up with a reasonable way to combine all of these into one spreadsheet. This should be done in Python. 

We should maintain important information from each separate index (identifiers, dates, source, if there's any location information, etc) but also should add a unified ID to the new spreadsheet that combines all of these. I'd like to add an ID for physical location in our warehouse as well.

**Comment by emilyodean** [comment]
[NVAerials2.xlsx](https://github.com/NBMG-UNR/resourcespace-migration/files/4093035/NVAerials2.xlsx)
[NVAerials3.xlsx](https://github.com/NBMG-UNR/resourcespace-migration/files/4093036/NVAerials3.xlsx)
[NVAerials4.xlsx](https://github.com/NBMG-UNR/resourcespace-migration/files/4093037/NVAerials4.xlsx)
[NVAerials7.xlsx](https://github.com/NBMG-UNR/resourcespace-migration/files/4093038/NVAerials7.xlsx)
[SCS_EXTRA_PHOTOS.xlsx](https://github.com/NBMG-UNR/resourcespace-migration/files/4093039/SCS_EXTRA_PHOTOS.xlsx)
[aerial_photos_index_2.xlsx](https://github.com/NBMG-UNR/resourcespace-migration/files/4093040/aerial_photos_index_2.xlsx)
[AERIALPH.xlsx](https://github.com/NBMG-UNR/resourcespace-migration/files/4093041/AERIALPH.xlsx)
[AerialPhotoData21.xlsx](https://github.com/NBMG-UNR/resourcespace-migration/files/4093042/AerialPhotoData21.xlsx)

**Comment by emilyodean** [comment]
@lweeks20 this can be done after #3 and #1

**Comment by emilyodean** [comment]
@lweeks20 please work on this tomorrow. Information is provided above. Combine the given spreadsheets into one spreadsheet with common column names. The column names are detailed and basic code is started here: https://github.com/NBMG-UNR/resourcespace-migration/blob/master/clean_airphotos.

**Comment by emilyodean** [comment]
[NVAerialsUSGS03032020.xlsx](https://github.com/NBMG-UNR/resourcespace-migration/files/4290220/NVAerialsUSGS03032020.xlsx)

**Comment by emilyodean** [comment]
@lweeks20, we'll chat tomorrow morning but I also wanted to give you some written information on this ticket. We just got 11 boxes of air photos from USGS which are at the top of the back stairs at GBSSRL. The spreadsheet above corresponds to the boxes, and each box has info written on it about its contents and which sheet within the spreadsheet contains the corresponding metadata. 

I need you to go through this spreadsheet and relate it to the master air photos sheet that you generated from all of disparate spreadsheets that we had. We need to identify if they sent us duplicates of any that we already have in our collection, and if there are duplicates we need to identify which duplicate is in better shape. If there are new air photos that we don't yet have in our collections, we need to separate those out so we can formally accession them. 

I'm going to leave the process of how to do the above up to you, but I would like an update at the end of each workday, so I'd recommend making a spreadsheet of which boxes you've gone through, which air photos are duplicates, and which ones are unique.

**Comment by lweeks20** [comment]
Here's our master list
[updated_aerial_photos_02132020.xlsx](https://github.com/NBMG-UNR/resourcespace-migration/files/4294175/updated_aerial_photos_02132020.xlsx)
